resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -27,6 +27,8 @@ ARG spark_uid=185
 # of the Spark distribution. E.g.:
 # docker build -t spark:latest -f kubernetes/dockerfiles/spark/Dockerfile .
 
+COPY kubernetes/dockerfiles/spark/sources.list /etc/apt/sources.list
```

Comment:
Can you 'inline' this into the Dockerfile rather than manage it as a separate file?